src/vmspawn/vmspawn-util.c

```diff
@@ -367,7 +367,7 @@ int find_qemu_binary(char **ret_qemu_binary) {
         return find_executable(qemu_arch_specific, ret_qemu_binary);
 }
 
-int vsock_fix_child_cid(unsigned *machine_cid, const char *machine, int *ret_child_sock) {
+int vsock_fix_child_cid(int vsock_fd, unsigned *machine_cid, const char *machine) {
```

Comment:
can you name this `vhost_vsock-fd` or so? i.e. to emphasize that this is the fd to `/dev/vhost_vsock` and not for example just an AF_VSOCK socket. Both are fds after all and used here, hence i think it's important to be precise to make this easy to read and follow.

Reply:
I've renamed it to vhost_device_fd so that it matches the name I use for this elsewhere.